### ISSUE TITLE
v1.9.1 seems not to be compatible with docker-gen latest (0.7.4)

### ISSUE THREAD

**Issue by BenKoerber** [JrCs/docker-letsencrypt-nginx-proxy-companion#460]
Hi, 

the current version (v1.9.1) of docker-letsencrypt-nginx-proxy-companion seems not to be compatible with docker-gen (latest 0.7.4). 

Using the docker-compose.yml shown below, I see that docker-gen exits with code 2:

```
docker@dcvm3:~/docker/app-config/reverse-proxy-container$ docker-compose up
Creating rp-nginx ... done
Creating rp-letsencrypt ... done
Creating rp-dockergen   ... done
Attaching to rp-nginx, rp-letsencrypt, rp-dockergen
rp-letsencrypt    | Info: Custom Diffie-Hellman group found, generation skipped.
rp-letsencrypt    | Reloading nginx docker-gen (using separate container rp-dockergen)...
rp-letsencrypt    | Reloading nginx (using separate container rp-nginx)...
rp-letsencrypt    | 2018/10/23 12:13:28 Generated '/app/letsencrypt_service_data' from 3 containers
rp-letsencrypt    | 2018/10/23 12:13:28 Running '/app/signal_le_service'
rp-letsencrypt    | 2018/10/23 12:13:28 Watching docker events
rp-letsencrypt    | 2018/10/23 12:13:28 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/signal_le_service'
rp-letsencrypt    | Sleep for 3600s
rp-letsencrypt    | 2018/10/23 12:13:28 Received event die for container edd204c4e130
rp-dockergen exited with code 2
```

If I use the image jrcs/letsencrypt-nginx-proxy-companion:v1.9 however, the containers come up nicely.
The obvious assumption is that the latest versions of jrcs/letsencrypt-nginx-proxy-companion and jwilder/nginx-proxy are not compatible. However, I would not know how to debug this any further.

Maybe you have a chance to look into this. 

Thanks
Ben.

```
# docker-compose.yml -> error with version v1.9.1
version: '2'
services:
  rp-nginx:
    image: nginx
    container_name: rp-nginx
    ports:
      - "80:80"
      - "443:443"
    volumes:
      - ./etc/nginx/conf.d:/etc/nginx/conf.d
      - ./etc/nginx/vhost.d:/etc/nginx/vhost.d
      - ./../../app-data/reverse-proxy-container/usr/share/nginx/html:/usr/share/nginx/html
      - ./../../app-data/reverse-proxy-container/etc/nginx/certs:/etc/nginx/certs:ro
      - ./../../app-data/reverse-proxy-container/etc/nginx/htpasswd:/etc/nginx/htpasswd:ro

  rp-dockergen:
    image: jwilder/docker-gen
    container_name: rp-dockergen
    command: -notify-sighup rp-nginx -watch /etc/docker-gen/templates/nginx.tmpl /etc/nginx/conf.d/default.conf
    volumes_from:
      - rp-nginx
    volumes:
      - /var/run/docker.sock:/tmp/docker.sock:ro
      - ./etc/docker-gen/templates/nginx.tmpl:/etc/docker-gen/templates/nginx.tmpl:ro

  rp-letsencrypt:
    image: jrcs/letsencrypt-nginx-proxy-companion:v1.9.1
    container_name: rp-letsencrypt
    environment:
      # COMMENT ACME_CA_URI for production use
      ACME_CA_URI: https://acme-staging.api.letsencrypt.org/directory
      NGINX_PROXY_CONTAINER: rp-nginx
      NGINX_DOCKER_GEN_CONTAINER: rp-dockergen
    volumes_from:
      - rp-nginx
    volumes:
      - ./../../app-data/reverse-proxy-container/etc/nginx/certs:/etc/nginx/certs:rw
      - /var/run/docker.sock:/var/run/docker.sock:ro

networks:
  default:
    external:
      name: rp-nginx-net
```

**Comment by buchdag** [comment]
Hi. Your `docker-gen` container dying has probably little (if any) to do with the LE companion container or its version.

I've been running `v1.9.1` and `jwilder/docker-gen:0.7.4` built from git since I tagged `v1.9.1`, without issue. I just tried again on another host with a `docker-gen` container built from the master branch of the git repo, no issues, no container exiting without reason.

@blackandred seems to have experienced something similar in #448
